### ISSUE TITLE
url error

### ISSUE THREAD

**Issue by wrecode** [cztomczak/cefpython#273]
in linux. if url is file:///home/xxx.html?param=1
CreateBrowserSync this url.
the url format to file:///home/xxx.html%3Fparam%3D1
can't open this url.

**Comment by cztomczak** [comment]
Url is being modified via GetNavigateUrl: https://github.com/cztomczak/cefpython/blob/cefpython54/src/utils.pyx#L85

Seems that urllib_pathname2url() function is causing this issue. There was already a fix for "#" character being encoded, a similar solution for "?" and "=" should be applied:
```
        # Allow hash when loading urls. The pathname2url function
        # replaced hashes with "%23" (Issue 114).
        url = url.replace("%23", "#")
```

Related issue: #114

**Comment by cztomczak** [comment]
Fixed in bfc0a2504ed07235dd62181a05551c86c7b6361d. v55.2 released, please test.
